src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -138,7 +138,8 @@ def determineOutputModules(self, scenarioFunc=None, scenarioArgs=None,
                                       'SingleTrack': 'AOD',
                                       'MinBias': 'AOD',
                                       'OniaUPC': 'RAW-RECO',
-                                      'HighMET': 'RECO'
+                                      'HighMET': 'RECO',
+                                      'BPHSkim': 'USER'
                                      }
                     dataTier = skimToDataTier.get(physicsSkim, 'USER')
```

Comment:
Why isn't this line sufficient for what you are doing in line 142? Also, a little trick, if you add a comma to the end of line 142 it does not impact the python but if you then need to add another skim later, you don't need to "delete" a line.